### DISCUSSION TITLE
MediaElement - Ability to integrate into system audio playback (notifications/lock screen)

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [CommunityToolkit/Maui#952] [New Feature Discussions]
From what I can tell at least on Android there is no integration into the notification system for playback controls. This was a nice feature we use in the .NET podcast app.

I haven't tested on iOS yet. Windows does a generic playback control on lock screen, but having control would be nice

**Comment by TheCodeTraveler** [comment]
Hey James! If I'm understanding correctly, this is something the .NET podcast app is already implementing which is great news! 

Could you share a link pointing to the that implementation in source code in that app? What would the level of difficulty be to port it to `CommunityToolkit.Maui`?

**Reply by jamesmontemagno** [reply]
It is a bit of code and dealing with remote input from the notification as well. https://github.com/microsoft/dotnet-podcasts/tree/main/src/Lib/SharedMauiLib/Platforms/Android 

Difficulty... Medium? 7/10?

**Comment by Mashedpoturtles** [comment]
I'd really appreciate this feature as well

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
